src/main.cc

```diff
@@ -117,7 +117,11 @@ int daemon_main_(LiteClient& client, uint64_t interval, bool return_on_sleep) {
 
   std::shared_ptr<LiteClient> client_ptr{&client, [](LiteClient* /*unused*/) {}};
   AkliteClientExt akclient{client_ptr, false, true};
-  akclient.CompleteInstallation();
+  auto finalize_result = akclient.CompleteInstallation();
+  if (finalize_result.status == InstallResult::Status::NeedsCompletion) {
+    LOG_ERROR << "A system reboot is required to complete a previous installation";
```

Comment:
I would use the terms users have already got used, i.e. "... required to finalize the pending installation."

Reply:
Done.